File: 217.contains-duplicate.py
#
# @lc app=leetcode id=217 lang=python3
#
# [217] Contains Duplicate
#
from typing import List
# @lc code=start
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        if len(set(nums)) != len(nums):
            return True
        return False
# @lc code=end
'''
time: O(n)
space: O(n)
'''

File: 217.contains-duplicate.js
/*
 * @lc app=leetcode id=217 lang=javascript
 *
 * [217] Contains Duplicate
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
  var set = new Set(nums);

  if (set.size === nums.length) {
    return false;
  }
  return true;
};
// @lc code=end

